apps/web/content/docs/components/hr.mdx

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
This file should be also moved to https://github.com/themesberg/flowbite-react/tree/main/apps/web/content/docs/typography

Reply:
Fixed!.